src/test/java/com/github/streams/practice/numbers/problems/I_SegregateEvenOddNumbers.java

```diff
@@ -16,10 +16,10 @@ class I_SegregateEvenOddNumbers {
   @Test
   @Disabled
   void testSegregationOfEvenOddNumbersTest() {
-    final var input = IntStream.range(1, 50).boxed();
-    final var mySolution = NumbersProblemSolution.segregateEvenOddNumbers(input);
+    final var input = IntStream.range(1, 50).boxed().toList();
```

Comment:
Using a terminal operation here appears unnecessary and might result in slow down of unit test case performance. We can streamline this code by keeping it within the Stream API.